ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/TLSv1_3IT.scala

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import com.daml.ledger.api.testtool.infrastructure.Allocation.{NoParties, allocate}
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.api.testtool.infrastructure.participant.ParticipantTestContext
+import com.daml.ledger.api.tls.TlsVersion.TlsVersion
+import com.daml.ledger.api.tls.{TlsConfiguration, TlsVersion}
+import com.daml.ledger.api.v1.ledger_identity_service.LedgerIdentityServiceGrpc.LedgerIdentityServiceBlockingStub
+import com.daml.ledger.api.v1.ledger_identity_service.{
+  GetLedgerIdentityRequest,
+  LedgerIdentityServiceGrpc,
+}
+import com.daml.ledger.resources.{ResourceContext, ResourceOwner}
+import io.grpc.netty.{NegotiationType, NettyChannelBuilder}
+import io.grpc.{Channel, StatusRuntimeException}
+
+import java.util.concurrent._
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+
+final class TLSv1_3IT extends LedgerTestSuite {
+
+  testTlsConnection(clientTls = TlsVersion.V1_3, assertion = assertSuccessfulConnection)
+  testTlsConnection(clientTls = TlsVersion.V1_2, assertion = assertFailedlConnection)
+  testTlsConnection(clientTls = TlsVersion.V1_1, assertion = assertFailedlConnection)
+  testTlsConnection(clientTls = TlsVersion.V1, assertion = assertFailedlConnection)
+
+  def testTlsConnection(clientTls: TlsVersion, assertion: Try[String] => Try[Unit]): Unit = {
+    test2(
+      "ConnectionOnTLSv1.3",
+      "A ledger API server accepts TLS 1.3 connection",
+      allocate(NoParties),
+    ) { implicit ec => (testContexts: Seq[ParticipantTestContext]) =>
+      { case _ =>
+        assume(testContexts.nonEmpty, "At least one context expected!")
+        val firstTextContext = testContexts.head
+        assume(
+          firstTextContext.clientTlsConfiguration.isDefined,
+          "This test case requires TLS configuration!",
+        )
+        val tlsConfiguration = firstTextContext.clientTlsConfiguration.get
+
+        val concurrency = Concurrency(
```

Comment:
(Thinking aloud..)
Appreciate hints as for what values would be sensible here.
Unless this config could be simplified away completely